### PR TITLE
SPM beta docs

### LINKED ISSUE #3136: FR: Support for Swift Package Manager
With iOS support for Swift Package Manager  in Xcode 11 can we please get that added to this project.

### PR DESCRIPTION
**Pull request by paulb777**
Fix #3136 

To review, see https://github.com/firebase/firebase-ios-sdk/blob/pb-spm-beta/SwiftPackageManager.md

### CODE REVIEW

File: SwiftPackageManager.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+in a format like 6.31-spm-beta. We won't support standard repository
+versioning until later in the beta or with general availability of the SPM
+distribution.
+
+<img src="docs/resources/SPMSelect.png">
+
+Choose the Firebase product and any additional products that you want installed
+in your app.
+
+<img src="docs/resources/SPMChoose.png">
```
Comment (ryanwilson) [inline SwiftPackageManager.md:41]: This should be `SPMProducts` not `SPMChoose`

File: SwiftPackageManager.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+<img src="docs/resources/SPMAddPackage.png">
+
+Select the Firebase GitHub repository:
+
+<img src="docs/resources/SPMChoose.png">
+
+Select the beta branch.
+
+Note: Starting with the 6.31.0 release, the versions are specified
+in a format like 6.31-spm-beta. We won't support standard repository
```
Comment (ryanwilson) [inline SwiftPackageManager.md:32]: Should we be using `6.31.0-spm-beta` (including the patch version)?

File: SwiftPackageManager.md (hunk 1 above)
Comment (paulb777) [reply SwiftPackageManager.md]: Good catch! Thanks!

File: SwiftPackageManager.md (hunk 2 above)
Comment (paulb777) [reply SwiftPackageManager.md:32]: I'm proposing that we update the 6.31-spm-beta branch with any patches.

**Pull request merged**